nncf/common/quantization/quantizer_propagation/solver.py

```diff
@@ -258,6 +386,9 @@ def __init__(self,
           trainable quantizer parameters.
         :param run_consistency_checks: Whether to run internal consistency checks at each propagataion step.
         :param quantize_outputs: Whether to insert additional quantizers right before each of the model outputs.
+        :param post_processing_marker_metatypes: The framework specific NNCF Metatypes, which are markers for
+        the model post-processing part. They are used for automatic ignoring post-processing nodes.
```

Comment:
Describe a little more specific the logic according to which the post-processing nodes are determined based on the marker metatypes.

Reply:
Done